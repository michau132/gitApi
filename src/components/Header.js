import React from 'react';
import styled from 'styled-components';

const H1 = styled.h1`
  font-size: 40px;
  color: #fdc22a;
  text-transform: uppercase;
  margin-left: 40px;
  span {
    color: #b6b6b6;
  }
`;

const Header = () => (
  <H1>
      beer
    <span>guru</span>
  </H1>
);

export default Header;
