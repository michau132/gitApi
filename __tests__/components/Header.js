import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from '../../src/components/Header';

describe('<Header /> component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Header />);
  });

  test('renders without crashing', () => {
    expect(wrapper).toBeDefined();
  });

  test('matching snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
