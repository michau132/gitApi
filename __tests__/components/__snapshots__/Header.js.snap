// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> component matching snapshot 1`] = `
<styled.h1>
  beer
  <span>
    guru
  </span>
</styled.h1>
`;
